src/metrics/include/metrics/MetricsReporter.hpp

```diff
@@ -295,6 +305,12 @@ class ReporterT final {
                         out << event.second.ops << ",";
                         out << event.second.errors << ",";
                         out << event.second.size << std::endl;
+
+                        if (++iter % 1000000 == 0 && logMode != LogMode::kNone) {
```

Comment:
Can we try adding another two 0s here? I would expect this loop to be at least a few dozen times faster than the loop that [writes out metrics to a file](https://github.com/mongodb/genny/blob/0b30c1585fd4b31dc5232e2bfa8e79473dbc3b7a/src/python/genny/parsers/cedar.py#L206) where we use 1e6